android/src/main/java/com/reactnativecommunity/webview/RNCWebViewClient.java

```diff
@@ -312,6 +380,19 @@ protected WritableMap createWebViewEvent(WebView webView, String url) {
         return event;
     }
 
+  protected WritableMap createWebViewEventTest(WebView webView, String url) {
```

Comment:
This was created because `createWebviewEvent` returns an error related to threads when setting one of the properties, will refactor this function